dozer-spring-boot-auto-configuration/dozer-spring-boot-autoconfigure/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2005-2017 Dozer Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.github.dozermapper</groupId>
+        <artifactId>dozer-spring-boot-auto-configuration</artifactId>
+        <version>6.2.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>dozer-spring-boot-autoconfigure</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Spring Boot AutoConfiguration :: Dozer AutoConfiguration</name>
```

Comment:
Can you update name to match other modules, i.e.: Dozer :: Module :: Sub Module
- https://github.com/DozerMapper/dozer/blob/master/spring/pom.xml#L33